pkg/server/config/manager.go

```diff
@@ -64,6 +65,32 @@ func NewManager(config MultusConf) (*Manager, error) {
 	return newManager(config, defaultPluginName)
 }
 
+// atomicFileWrite writes our configs atomically to avoid partial reads
+func atomicFileWrite(filename string, data []byte, perm os.FileMode) error {
```

Comment:
As warning mentiond, perm is not used in the function hence, none changed the files permission to 0644. Could you please change the file permission?